### ISSUE TITLE
CAM011 King - Bow ball etc...

### ISSUE THREAD

**Issue by tristanjph2** [13enedict/CCRC#60]
Bow ball missing needs replacing before learn to row next weekend.
The rudder wire needs adjusting, and white plastic parts need moving so they are level when the rudder is straight.

**Comment by 13enedict** [comment]
I've adjusted the steering, I had to leave quite a bit of slack because, for some reason, it runs against the cox's torso. 

However, it looks like the club has burned its way through all the spare bow balls we had. There were about 8-10 and I thought that was an excessive number. Unbelievable. I'll order some more but they won't be ready by the weekend so King is out of action and must not be used. 

I'll email Phil who has it booked to tell him not to use it.
